v8env_test/proxy.spec.js

```diff
@@ -3,12 +3,22 @@ import { expect } from 'chai'
 import * as proxy from '@fly/proxy'
 
 const origin = "https://fly.io/proxy/"
-const req = new Request("https://wat.com/path/to/thing", { headers: { "host": "wut.com" } })
+const req = new Request("https://wat.com/path/to/thing", { headers: { "host": "wat.com" } })
```

Comment:
This threw me for a loop at first. But I think the `host` value here is incorrect for a typical request.

Reply:
That `wat/wut` distinction is probably too silly. It was meant to test that the breq host header matched the one defined in the header options, not the hostname from the URL.